Resources/Prototypes/Entities/Structures/Power/Generation/solar.yml

```
@@ -58,7 +58,7 @@
   id: SolarPanelPlasma
   parent: SolarPanelBasePhysSprite
   name: solar panel plasma
-  description: A plasma solar panel that generates power.
+  description: A few sheets of plasma glass, that genarate electricity when photons hit them.
```

```suggestion
  description: A few sheets of plasma glass that generate electricity when hit by photons.
```
Typo, comma is unnecessary.